### DISCUSSION TITLE
Can comments be disabled for specific pages?

### DISCUSSION THREAD

**Discussion by abeebyekeen** [HEIGE-PCloud/DoIt#713] [Q&A] [answered]
Can the comment box be disabled for specific posts while still active for other posts?

**Comment by HEIGE-PCloud** [comment, marked as answer]
Yes, set 

```
comment:
  enable: false
```

in the front matter will disable comment for that page.

**Reply by abeebyekeen** [reply]
Awesome!
Thanks greatly for the swift response!!

**Reply by CXwudi** [reply]
So is it true that any settings under `[params.page]` can be customized for a specific page?

**Reply by HEIGE-PCloud** [reply]
Yes
